core/Symbols.h

```diff
@@ -103,6 +103,10 @@ class Symbol final {
 
     Loc loc() const;
     const InlinedVector<Loc, 2> &locs() const;
+    inline InlinedVector<Loc, 2> &locs() {
+        return locs_;
+    }
```

Comment:
Sorbet intentionally doesn't provide mutable access to `locs_`, because it wants to enforce that all new locs are added via `Symbol::addLoc` (defined below).

I think you can get rid of this function if you change your `auto loc` to `const auto &loc` below.